### DISCUSSION TITLE
While creating resource, the feature link to next slide on click is not working

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#600] [Bugs] [answered]
I'm working on portal 4.6 version.
While creating the learning resource, the feature of link to next slide on click is not working. 
Getting an error message of **command failed.** 
Attaching the screenshot below
![image](https://github.com/Sunbird-Ed/Community/assets/107101132/ba469e3d-714f-489e-afda-4206050a9d50)

@rajeevsathish @Krishnaj20 @vinukumar-vs

**Comment by Krishnaj20** [comment]
@nidhikachhawa24 - Pls share sufficient information such as logs at the least for the community to look into this issue. Thanks

@amitpriyadarshi

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - any update on the info that we have requested?

@amitpriyadarshi

**Reply by nidhikachhawa24** [reply]
@Krishnaj20 
Yes we're looking into logs, will provide you soon

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - Pls do check with @vinod303 . He should be able to advise.

**Reply by nidhikachhawa24** [reply]
@Krishnaj20 
Not getting any errors in the network, so not able to get the logs.

**Comment by vinod303** [comment, marked as answer]
@nidhikachhawa24 
1. When i tried, it is working as expected.
2. Please try saving the content once, close the editor and open it again i try.
3. Also, try with a newly uploade image

**Reply by nidhikachhawa24** [reply]
@vinod303 
It is sometimes not working even after saving the content once.

**Comment by vinod303** [comment]
@nidhikachhawa24  lets have a call to see this

**Reply by vinod303** [reply]
@nidhikachhawa24  can we close this ?

**Reply by nidhikachhawa24** [reply]
yes
@vinod303

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - Pls update the resolution and close this discussion. Thanks.

@vinod303

**Reply by Krishnaj20** [reply]
@vinod303 - Can you get the resolution details updated please so that community can benefit?

**Reply by vinod303** [reply]
@Krishnaj20  the user is not facing the issue now.. What user did was  to remove the slide and then re-created the slide again then added the transition. It worked post that it seems. Currently do not know what was causing the issue as the issue does not exist.  If it re-appears again, i will debug the same.
